### PR TITLE
Use T411 API instead of web scrapping

### PR DESCRIPTION
**Pull request by vanackej**
Use T411 API instead of web scrapping

### CODE REVIEW

File: sickbeard/providers/t411.py [hunk 1]
```diff
@@ -48,14 +48,16 @@ def __init__(self):
...
+        self.tokenLastUpdate = None
 
         self.cache = T411Cache(self)
 
         self.urls = {'base_url': 'http://www.t411.io/',
-                'search': 'http://www.t411.io/torrents/search/?name=%s&cat=210&subcat=%s&search=%s&submit=Recherche',
-                'login_page': 'http://www.t411.io/users/login/',
-                'download': 'http://www.t411.io/torrents/download/?id=%s',
-                }
+                     'search': 'https://api.t411.me/torrents/search/%s?cid=%s&limit=100',
```
Comment (nicolasmartinelli) [inline sickbeard/providers/t411.py]: Hi,
Why do you use t411.me instead of t411.io?

Comment (vanackej) [inline sickbeard/providers/t411.py]: The API is still available at api.t411.me, but you're right I should use the .io

**Comment by abeloin** [comment]
@vanackej Please rebase againt the latest version of the develop branch

**Comment by vanackej** [comment]
@abeloin Sorry, I did it this afternoon, but did not pushed the rebased branch. It is fixed now.

I don't think this PR should be qualified as bugfix, as this pull request does not fix anything broken. Using the API is just more efficient (my Synology slow CPU like it) and allows for more features to be implemented based on the T411 API.

**Comment by vanackej** [comment]
@abeloin Do you prefer if I squash my commits to one ?

**Comment by vanackej** [comment]
Squashed

**Comment by abeloin** [comment]
@vanackej Thanks

**Pull request merged**